app/services/reservations/create_reservation.rb

```diff
@@ -17,6 +17,8 @@ def call
         @reservation.save!
         create_tickets
       end
+
+      ReservationMailer.with(reservation: @reservation).reservation_created.deliver_later
```

Comment:
It works, but why do you create a [getter](https://www.geeksforgeeks.org/ruby-getters-and-setters-method/) (`attr_reader`) for `seats`, but you don't do it for `reservation`?
I'm curious. Cause it looks inconsistent to a reader.

Reply:
At the time I was thinking that i only need seats but now as you mentioned it I see that I should also add a reservation